scripts/spectrum-tokens.js

```diff
@@ -29,6 +29,7 @@ const tokensRoot = path.join(
     '*.css'
 );
 
+/** @todo Could generate this from CSS packages that have @spectrum-css/tokens as a dependency */
```

Comment:
The goal will be to not need an allow list here at some point, but would certainly be an interesting way to manage this data in the interim. We have used it as a human reabable collection of packages that have made the Core Tokens conversion to date, but hopefully we'll be through this process before too much longer!